package.json

```diff
@@ -94,7 +88,7 @@
         "diff": "^3.3.0",
         "eslint": "^4.19.1",
         "js-beautify": "^1.7.5",
-        "prettier": "^1.12.1",
+        "prettier": "^1.13.5",
```

Comment:
Please leave dependencies unrelated to this feature as they are.  We're working on updating them and keeping them up to date via Renovate.